### DISCUSSION TITLE
How to use ND_JUKEBOX_DEVICES environment variable ?

### DISCUSSION THREAD

**Discussion by b0bbywan** [navidrome/navidrome#3146] [Q&A] [answered]
I've seen the toml version in the doc here https://www.navidrome.org/docs/usage/jukebox/
But I'm unable to do this with an environment variable in docker-compose
I end up with this error `level=error msg="Fatal error in Navidrome. Aborting" error="audio device definition ought to contain 2 fields, found: 1 "` using this `ND_JUKEBOX_DEVICES: '[["pulse","pulse/alsa_output.platform-soc_sound.stereo-fallback"], ["auto", "auto"]]'`
If I remove the single quotes I get a docker compose validation error

**Comment by deluan** [comment]
I'm not sure if TOML arrays can be represented as env vars... Try to create a TOML config file for your installation: https://www.navidrome.org/docs/installation/docker/#:~:text=If%20you%20want%20to%20use%20a%20configuration%20file

**Reply by deluan** [reply, marked as answer]
Just tested and unfortunately it cannot be used as an env var. I fixed the docs to reflect that.
